pandas/tests/plotting/test_frame.py

```diff
@@ -231,14 +231,41 @@ def test_plot_xy(self):
     @pytest.mark.slow
     def test_logscales(self):
         df = DataFrame({'a': np.arange(100)}, index=np.arange(100))
+
         ax = df.plot(logy=True)
         self._check_ax_scales(ax, yaxis='log')
+        assert ax.get_yscale() == 'log'
```

Comment:
Rather than these individual checks can you parametrize all of the combinations?

Reply:
thanks! @WillAyd i parametrize the combinations.